controllers/v1/observationsController.js

```diff
@@ -631,128 +630,140 @@ module.exports = class Observations extends Abstract {
                 let messageData = messageConstants.apiResponses.ENTITY_FETCHED;
```

Comment:
@VISHNUDAS-tunerlabs to avoid memory usage, don't assign unwanted variables. please directly assign the value in the response if it is possible